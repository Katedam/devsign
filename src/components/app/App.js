import React from 'react';
import { 
  BrowserRouter as Router, 
  Route, 
  Switch 
} from 'react-router-dom';
import Signin from '../Signin';
import Signup from '../Signup.js';
import Home from '../Home';
import Profile from '../Profile';
import 'normalize.css';


export default function App() {
  return ( 
    <Router>
      <>
      <Switch>
        <Route path="/signin" component={Signin}></Route>
        <Route path="/signup" component={Signup}></Route>
        <Route path="/home" component={Home}></Route>
        <Route path="/profile" component={Profile}></Route>
      </Switch>
      </>
    </Router>
  );
}
