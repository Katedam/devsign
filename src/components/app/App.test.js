import React from 'react';
import App from './App';
import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom';

jest.mock('../../services/auth.js', () => ({}));

describe('App component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <MemoryRouter>
        <App />
      </MemoryRouter>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
