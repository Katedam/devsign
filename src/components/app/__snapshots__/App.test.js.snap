// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component matches a snapshot 1`] = `
Array [
  <header
    className="sc-bwzfXH jdLVsO"
  >
    <h1
      className="sc-htpNat hHwDue"
    >
      Chirp
    </h1>
  </header>,
  <form
    className="sc-ifAKCX coNtEK"
  >
    <label
      className="sc-EHOje kYkHpN"
      htmlFor="handle"
    >
      Chirp handle
    </label>
    <input
      className="sc-bZQynM hfwKph"
      id="handle"
      name="handle"
      readOnly={true}
      type="text"
      value="handle"
    />
    <label
      className="sc-EHOje kYkHpN"
      htmlFor="password"
    >
      password
    </label>
    <input
      className="sc-bZQynM hfwKph"
      id="password"
      name="password"
      readOnly={true}
      type="password"
      value="password"
    />
    <label
      className="sc-EHOje kYkHpN"
      htmlFor="sign-in"
    />
    <button
      className="sc-gzVnrw hkbkeK"
      id="sign-in"
    >
      Sign In
    </button>
    <a
      className="sc-bdVaJa cVFclJ"
      href="/signup"
      onClick={[Function]}
    >
      <p
        className="sc-htoDjs kQfXqe"
      >
        Not a User?
      </p>
    </a>
  </form>,
]
`;
